draft-abiggs-saag-key-management-service.md

```diff
@@ -1599,7 +1603,7 @@ JWE(K_ephemeral, {
 
 ### Delete Authorization
 
-To remove an authorization from a KMS resource object, any user currently authorized on the same resource object may issue a delete authorization request.  The request message conforms to the basic request message structure, where the method is "delete", and the URI is either that of the authorization object to be deleted, or the URI of the collection of authorizations within a particular KMS resource object appended with an authId query parameter whose value matches that of the authorization object to be deleted.
+To remove an authorization from a KMS resource object, any user currently authorized on the same resource object may issue a delete authorization request.  The request message conforms to the basic request message structure, where the method is "delete", and the URI is either that of the authorization object to be deleted, or a collection of authorizations within a particular KMS resource object appended to an authIds variable parameter whose value matches that of the authorization object to be deleted.
 
```

Comment:
I don't quite understand this phrase: "a collection of authorizations within a particular KMS resource object appended to an authIds variable parameter whose value matches that of the authorization object to be deleted."

Reply:
how about this:
```
a collection of authorizations, included as an attribute authIds to a particular KMS resource object, whose value matches that of the authorization object to be deleted.
```

How is this?

Reply:
Hm, still not clear.  Might be necessary to break into more than one sentence.